src/files.ts

```diff
@@ -88,106 +88,103 @@ export async function getProjectFiles(rootDir: string = path.join('.', '/'), cal
     const file2path: Array<{ path: string; file: AppsScriptFile }> = [];  // used by `filePushOrder`
     let ignoredFilePaths: string[] = [];
     ignoredFilePaths = ignoredFilePaths.concat(ignorePatterns);
-    // Match the files with ignored glob pattern
-    readMultipleFiles(filePaths, 'utf8', (err: string, contents: string[]) => {
-      if (err) return callback(new Error(err), null, null);
-      // Check if there are files that will conflict if renamed .gs to .js.
-      // When pushing to Apps Script, these files will overwrite each other.
-      filePaths.map((name: string) => {
-        const fileNameWithoutExt = name.slice(0, -path.extname(name).length);
-        if (
-          filePaths.indexOf(fileNameWithoutExt + '.js') !== -1 &&
-          filePaths.indexOf(fileNameWithoutExt + '.gs') !== -1
-        ) {
-          // Can't rename, conflicting files
-          abortPush = true;
-          if (path.extname(name) === '.gs') {
-            // only print error once (for .gs)
-            logError(null, ERROR.CONFLICTING_FILE_EXTENSION(fileNameWithoutExt));
-          }
+
+    // Check if there are files that will conflict if renamed .gs to .js.
+    // When pushing to Apps Script, these files will overwrite each other.
+    filePaths.map((name: string) => {
+      const fileNameWithoutExt = name.slice(0, -path.extname(name).length);
+      if (
+        filePaths.indexOf(fileNameWithoutExt + '.js') !== -1 &&
```

Comment:
Optional: Use `includes()` here:

```js
filePaths.includes(fileNameWithoutExt + '.js')
```

Reply:
Good refactor. Not going to do this in this PR, but we should remove `indexOf` everywhere:

https://github.com/google/clasp/search?q=indexOf&unscoped_q=indexOf